.config/CredScanSuppressions.json

```diff
@@ -0,0 +1,33 @@
+{
+    "tool": "Credential Scanner",
+    "suppressions": [
+        {
+            "file": "proxy_test.go",
+            "_justification": "sample login for testing"
+        },
+        {
+            "file": "pullsecret_test.go",
+            "_justification": "sample login for testing"
+        },
+        {
+            "file": "viper.go",
+            "_justification": "example login from comments"
+        },
+        {
+            "file": "README.md",
+            "_justification": "example login from comments"
+        },
+        {
+            "file": "api.go",
+            "_justification": "example secret in comment"
+        },
+        {
+            "file": "machine_webhook.go",
+            "_justification": "false positive"
+        },
+        {
+            "file": "doc.go",
+            "_justification": "examples in comments"
+        }
+    ]
+}
```

Comment:
Please set your code editor to insert `/n` at the end. I can see how linters will complain about this

Reply:
Thanks - this ended up being a part of the larger issue.

at the end of the test-python, there is a script that is run (hack/format-yaml/format-yaml.py).  This file was modifying the initial set of pipelines we created, which is what caused [[ -z "$(git status -s)" ]] to return a non-zero.  Part was due to the missing empty line at the end of the file, but also general formatting changes (no ' around strings, tabbing, etc).